packages/coinstac-common/src/models/pipeline/pipeline.js

```diff
@@ -97,64 +97,81 @@ class Pipeline extends Base {
   }
 
   /**
-   * @description run the pipeline from its current state, feeding in inputs
+   * Run.
+   * @description Run the pipeline from its current state, feeding in inputs
    * provided by the pipeline caller
-   * @param {object} runInputs object that is fed to the computation, provided by
-   *                      the pipeline runner
-   * @param {ComputationResult} compResult the ComputationResult for the environment.
-   *                                @note this should only be touched by plugins.
+   *
+   * @param {Object} runInputs Input passed to the computation, provided by the
+   * pipeline runner.
+   * @param {ComputationResult} compResult Environment-specific
+   * ComputationResult. This should only be touched by plugins.
    * @returns {Promise}
    */
   run(runInputs, compResult) {
     if (!runInputs || !compResult) {
-      return Promise.reject(
-        new ReferenceError('options & ComputationResult required to pass to computations')
-      );
-    }
-    if (this.inProgress) {
-      return Promise.reject(
-        new Error([
-          'pipelines do not permit concurrent running. please spawn a new',
-          'pipeline instance if concurrency is required',
-        ].join(' '))
-      );
+      return Promise.reject(new Error(
+        'Run inputs and ComputationResult required to pass to computations'
+      ));
+    } else if (this.inProgress) {
+      return Promise.reject(new Error(
+        /* eslint-disable max-len */
```

Comment:
When we update to the latest AirBnB styles we can drop the `max-len` disable. This is smart: makes it easier to grep/ag for strings in the codebase.